README

```diff
@@ -188,6 +188,16 @@ Compiler Notes
   for more details:
   https://github.com/open-mpi/ompi/issues/3612
 
+- Compiling Fortran programs using the mpi_f08 module on PowerPC with
+  the PGI (tested 17.5) or XL (tested v15.1.5) Fortran compilers and GNU
+  linker after 2.25.1 and before 2.28 will likely experience runtime failures.
```

Comment:
Per IM discussion with @jjhursey, I think that this above sentence is trying to say that compiling MPI applications that utilize the `mpi_f08` module with the ... compilers on the ... platforms with the GNU linker ... will likely experience runtime failures when running the MPI applications.  Even that language is a little awkward -- the key points here are:

1. compiling Fortran MPI applications that utilize `mpi_f08` (vs. compiling Open MPI itself)
1. problems occur when you run the MPI application (vs. when compiling or linking Open MPI and/or the application)